package.json

```diff
@@ -11,12 +11,11 @@
   },
   "scripts": {
     "build": "shx rm -rf dist && git rev-parse HEAD > .commithash && rollup -c && shx cp src/rollup/types.d.ts dist/rollup.d.ts && shx chmod a+x bin/rollup",
-    "build:test": "shx rm -rf dist && rollup -c --configNoBrowser && shx cp src/rollup/types.d.ts dist/rollup.d.ts && shx chmod a+x bin/rollup",
+    "build:test": "shx rm -rf dist && rollup -c --configTest && shx cp src/rollup/types.d.ts dist/rollup.d.ts && shx chmod a+x bin/rollup",
     "build:bootstrap": "bin/rollup -c && shx cp src/rollup/types.d.ts dist/rollup.d.ts && shx chmod a+x bin/rollup",
-    "ci:coverage": "npm run test:coverage",
     "ci:lint": "npm run lint:nofix && npm run security",
     "ci:test": "npm run build:test && npm run build:bootstrap && npm run test:all",
-    "ci:test_12": "npm run build:test && npm run build:bootstrap && npm run test:only && npm run test:typescript",
+    "ci:coverage": "npm run build:test && nyc --reporter lcovonly mocha && codecov",
```

Comment:
_(late to the party, but I'll submit a PR for this)_

We can eliminate the `codecov` dependency by adding this to the circle ci config on the last node version test:

```yml
      - run:
          name: Submit coverage data to codecov.
          command: bash <(curl -s https://codecov.io/bash)
          when: on_success
```